presidio-analyzer/Dockerfile.windows

```diff
@@ -3,7 +3,7 @@ FROM python:3.9-windowsservercore
 ARG NLP_CONF_FILE=presidio_analyzer/conf/default.yaml
 ARG ANALYZER_CONF_FILE=presidio_analyzer/conf/default_analyzer.yaml
 ARG RECOGNIZER_REGISTRY_CONF_FILE=presidio_analyzer/conf/default_recognizers.yaml
-ENV PIPENV_VENV_IN_PROJECT=1
+ENV PDM_VENV_WITH_PIP=1
```

Comment:
Is there a poetry equivalent?

Reply:
If the build passed, then it means we do not need an equivalent. 
pdm environments comes without pip, this installs it for pdm